jcl/src/java.base/share/classes/java/lang/Class.java

```diff
@@ -423,7 +425,9 @@ public static Class<?> forName(String className) throws ClassNotFoundException
 		callerClassLoader = caller.getClassLoaderImpl();
 	}
 	Class<?> c = forNameImpl(className, false, callerClassLoader);
+	/*[IF JAVA_SPEC_VERSION < 24]*/
```

Comment:
This isn't needed since it is within `/*[ELSE] JAVA_SPEC_VERSION >= 18`.

Reply:
Good catch, thanks.